--- track/web/app/src/Features/Project/ProjectEditorHandler.js	2024-12-05 15:27:44.219830779 +0000
+++ /docker/compose/overleafserver/data/to_mod/track/ProjectEditorHandler.js	2024-12-05 16:52:38.835879765 +0000
@@ -8,7 +8,7 @@
 }
 
 module.exports = ProjectEditorHandler = {
-  trackChangesAvailable: true,
+  trackChangesAvailable: false,
 
   buildProjectModelView(project, members, invites, deletedDocsFromDocstore) {
     let owner, ownerFeatures
@@ -57,7 +57,7 @@
       references: false,
       referencesSearch: false,
       mendeley: false,
-      trackChanges: true,
+      trackChanges: false,
       trackChangesVisible: ProjectEditorHandler.trackChangesAvailable,
       symbolPalette: false,
     })
