--- openid/services/web/app/views/layout/navbar-website-redesign.pug	2024-12-05 15:27:44.218830795 +0000
+++ /docker/compose/overleafserver/data/to_mod/openid/navbar-website-redesign.pug	2024-12-05 16:52:38.403884961 +0000
@@ -140,13 +140,13 @@
 						// login link
 						li.secondary
 							a(
-								href="/login/oidc"
+								href="/login"
 								event-tracking="menu-clicked-login"
 								event-tracking-action="clicked"
 								event-tracking-trigger="click"
 								event-tracking-mb="true"
 								event-segmentation={ page: currentUrl }
-							) #{translate("log_in")}
+							) #{translate('log_in')}
 
 					// projects link and account menu
 					if getSessionUser()
