--- openid/services/web/app/src/infrastructure/Server.js	2024-12-05 15:27:44.216830827 +0000
+++ /docker/compose/overleafserver/data/to_mod/openid/Server.js	2024-12-05 16:52:38.342885695 +0000
@@ -24,7 +24,7 @@
 const bearerTokenMiddleware = require('express-bearer-token')
 
 const passport = require('passport')
-const OpenIDConnectStrategy = require('passport-openidconnect').Strategy
+const LocalStrategy = require('passport-local').Strategy
 
 const oneDayInMilliseconds = 86400000
 const ReferalConnect = require('../Features/Referal/ReferalConnect')
@@ -207,18 +207,16 @@
 webRouter.use(passport.initialize())
 webRouter.use(passport.session())
 
-passport.use(new OpenIDConnectStrategy({
-      issuer: process.env.OIDC_ISSUER,
-      authorizationURL: process.env.OIDC_AUTHORIZATION_URL,
-      tokenURL: process.env.OIDC_TOKEN_URL,
-      userInfoURL: process.env.OIDC_USERINFO_URL,
-      clientID: process.env.OIDC_CLIENT_ID,
-      clientSecret: process.env.OIDC_CLIENT_SECRET,
-      callbackURL: process.env.OIDC_CALLBACK_URL,
-      scope: 'openid profile email',
+passport.use(
+  new LocalStrategy(
+    {
+      passReqToCallback: true,
+      usernameField: 'email',
+      passwordField: 'password',
     },
-    AuthenticationController.verifyOpenIDConnect));
-
+    AuthenticationController.doPassportLogin
+  )
+)
 passport.serializeUser(AuthenticationController.serializeUser)
 passport.deserializeUser(AuthenticationController.deserializeUser)
 
