--- track/web/config/settings.defaults.js	2024-12-05 15:27:44.220830763 +0000
+++ /docker/compose/overleafserver/data/to_mod/track/settings.defaults.js	2024-12-05 16:52:38.894879055 +0000
@@ -113,11 +113,6 @@
 module.exports = {
   env: 'server-ce',
 
-  ldap: {
-    enable: true,
-    updateUserDetailsOnLogin: true
-  },
-
   limits: {
     httpGlobalAgentMaxSockets: 300,
     httpsGlobalAgentMaxSockets: 300,
@@ -907,7 +902,6 @@
     'launchpad',
     'server-ce-scripts',
     'user-activate',
-    'track-changes',
   ],
   viewIncludes: {},
 
